README.md

```diff
@@ -187,3 +187,101 @@ After that you can start QEMU
 ```shell
 qemu-system-x86_64 -serial stdio -drive file=trenchboot.img,if=virtio -enable-kvm
 ```
+
+## Development
+
+Usually after making changes to recipes it's enough to just run [build](#build)
+again but sometimes not all changes are detected. In those cases you can clean
```

Comment:
> but sometimes not all changes are detected

Any specific examples of such scenario?
If this is the case, we are celarly doing something wrong.

Reply:
Not from the top of my head, at least not in this project. I'm sure it happened in different projects but probably on older Yocto versions.

Reply:
So let's not confuse readers here if we have no such examples.

Reply:
Fixed in 4d26e2f0d7a3cfe949593e8a06c2265f3ff94ac9